website/ocular-config.js

```diff
@@ -58,7 +59,7 @@ module.exports = {
 
   // TODO/ib - from gatsby starter, clean up
   siteUrl: "https://ocular", // Domain of your website without pathPrefix.
-  pathPrefix: "/math", // Prefixes all links. For cases when deployed to example.github.io/gatsby-advanced-starter/.
+  pathPrefix: "/math.gl/", // Prefixes all links. For cases when deployed to example.github.io/gatsby-advanced-starter/.
   siteRss: "/rss.xml", // Path to the RSS file.
```

Comment:
I think only the first two or three of these legacy settings from the original forked starter are used. We should replace all of these with all CAPS constants in the same style as the rest of the config. These legacy settings have been confusing new users of ocular...